Moonshot/baddies/BaddieLaser/BaddieLaserPointer.gd

```diff
@@ -33,7 +33,7 @@ func shoot_player(player_global_position: Vector2) -> void:
 	var shot = BaddieBullet.instance()
 	shot.set_damage(damage)
 	
-	get_tree().get_root().add_child(shot)
+	Utils.Player.get_tree().get_root().add_child(shot)
```

Comment:
? this feels weird

Reply:
Feels weird yeah, but makes sense as the player is always going to be in the scene tree to be able to fetch it, it is a bug fix bundled in with this PR